vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -1210,7 +1211,8 @@ public void onClick(DialogInterface d, int n) {
                                     createNewChat(VectorRoomCreationActivity.RoomCreationModes.DIRECT_CHAT);
                                 } else if (1 == n) {
                                     // Create an empty room
```

Comment:
Could you please update the comment here?
BTW the method `createNewRoom` can be removed from VectorHomeActivity